Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -94,6 +94,7 @@ namespace WebCore {
 
 WTF_MAKE_TZONE_OR_ISO_ALLOCATED_IMPL(HTMLInputElement);
 
+using namespace CSS::Literals;
```

Comment:
Noticed this on the original pull request: too bad that we need “CSS“ both in the namespace path and in the suffix for the literal syntax. Seems that one or the other would suffice.

Reply:
I think the one to remove it from would be the literals themselves and I am coming around to that.